documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -6,11 +6,15 @@ The prompt and accurate execution of liquidations and disputes is a core assumpt
 Liquidation and dispute bots, as described below and implemented [here](https://github.com/UMAprotocol/protocol/tree/master/liquidator) and [here](https://github.com/UMAprotocol/protocol/tree/master/disputer), are infrastructure tools that will help maintain the overall health of the UMA ecosystem.
 They are currently compatible with the priceless synthetic token contract template, as described [here](../explainer.md) and implemented [here](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/financial-templates).
 
+### Liquidation vs Dispute Bot
+
 The liquidation bot monitors all open positions within a given expiring multi-party contract and liquidates positions if their collateralization ratio, as inferred from off-chain information about the value of the price identifier, drops below a given threshold.
 
 The dispute bot monitors all liquidations occurring within a given expiring multi-party contract and initiates disputes against liquidations it deems invalid, as inferred from off-chain information about the value of the price identifier.
 A liquidation is invalid if a position was correctly collateralized at the time of liquidation.
 
+In short, a liquidation bot is to liquidate under-collateralized positions while a dispute bot is used for disputing inaccuracies within the UMA DVM system. Only a dispute triggers a price request in the system, so disputes are very important to the operation of the DVM.
```

Comment:
I think the explanation of dispute bots is a little off. I would suggest something like the following:

In short, a liquidation bot is to liquidate under-collateralized positions while a dispute bot is used for disputing those liquidations if they incorrectly report a position as under-collateralized. Only a dispute triggers a price request in the system, so disputes are very important to the operation of the financial contract.

Reply:
Great, I'll integrate your suggestion.

btw @mrice32 if you use the "Insert a suggestion" button, I can hit a simple button and Github will automatically make a commit to integrate that change.

<img width="834" alt="Screen Shot 2020-06-23 at 1 51 27 PM" src="https://user-images.githubusercontent.com/943555/85437542-bbc1a100-b558-11ea-9ed9-261f1c73ea10.png">

Reply:
Yup, I'm familiar. I only use that when I think my suggestion should be taken word-for-word. In this case, I figured you might want to toy with the language a little more, so I left as a regular comment.